numba_dpex/core/dpex_exceptions.py

```diff
@@ -32,3 +33,164 @@ def __init__(self, kernel_name, return_type) -> None:
         )
 
         super().__init__(self.message)
+
+
+class InvalidKernelLaunchArgsError(Exception):
+    """Raised when a kernel is dispatched with insufficient or incorrect launch
+    arguments.
+
+    A global range value is needed to submit a kernel to an execution queue. The
+    global range provides the number of work items that the SYCL runtime should
+    create to execute the kernel. The exception is raised if a kernel is
+    dispatched without specifying a valid global range.
+
+    Args:
+        kernel_name (str): The kernel function name.
+    """
+
+    def __init__(self, kernel_name, *args):
+
+        if not args:
+            self.message = (
+                f"No global range specified for launching the Kernel "
+                f"{kernel_name}. "
+                f"A valid global range tuple is needed to launch a kernel."
```

Comment:
```suggestion
            self.message = (
                "No global range specified for launching the Kernel "
                f"{kernel_name}. "
                "A valid global range tuple is needed to launch a kernel."
```

Linters would complain of use of format string but no template argument